pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -732,7 +731,12 @@ func (r *Builder) PopulatorVolumes(vmRef ref.Ref, annotations map[string]string,
 				return
 			}
 			var pvc *core.PersistentVolumeClaim
-			storageClassName := r.Context.Map.Storage.Spec.Map[0].Destination.StorageClass
+			if sdToStorageClass == nil {
+				if sdToStorageClass, err = r.mapStorageDomainToStorageClass(); err != nil {
+					return
+				}
```

Comment:
why not pulling it out before the loop?

Reply:
because it could be that we won't need this, e.g., for VMs with a single LUN